src/components/Dashboard/Dashboard.spec.jsx

```diff
@@ -3,7 +3,7 @@ import { act, render, fireEvent, screen } from "@testing-library/react";
 
 import Dashboard from "./Dashboard";
 
-describe("Dashboard", () => {
+describe.skip("Dashboard", () => {
```

Comment:
we can remove it now... identified the issue why the github test action was failing

Reply:
removed skip